### PR TITLE
Add recipe for modernize 0.5

### PR DESCRIPTION
**Pull request by nicoddemus**

### CODE REVIEW

File: recipes/modernize/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  imports:
+    - libmodernize
+    - libmodernize.fixes
+
+  commands:
+    - python-modernize --help
+
+about:
+  home: https://github.com/python-modernize/python-modernize
+  license: BSD
```
Comment (synapticarbors) [inline recipes/modernize/meta.yaml]: Change to `BSD 3-clause`

File: recipes/modernize/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  license_file: LICENSE
+  license_family: BSD
+  summary: 'A hack on top of 2to3 for modernizing code for hybrid codebases.'
```
Comment (synapticarbors) [inline recipes/modernize/meta.yaml]: Add:

```
dev_url: https://github.com/python-modernize/python-modernize
doc_url: http://python-modernize.readthedocs.io/
```

File: recipes/modernize/meta.yaml (hunk 1 above)
Comment (nicoddemus) [inline recipes/modernize/meta.yaml]: Done, thanks

File: recipes/modernize/meta.yaml (hunk 2 above)
Comment (nicoddemus) [inline recipes/modernize/meta.yaml]: Done, thanks

**Pull request merged**